lib/facter/lvm_support.rb

```diff
@@ -4,8 +4,8 @@
   confine :kernel => :linux
 
   setcode do
-    vgdisplay = Facter::Util::Resolution.which('vgs')
-    vgdisplay.nil? ? nil : true
+    `which vgs`
+    $?.success? ? true : nil
```

Comment:
Hm, this should matter? In the original it's checked on nil and returns 'true' when not nil. The new one checks on success and returns 'true' when success ist true. Both cases return true under the same conditions?